examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -485,6 +270,40 @@ void ApplicationInit()
     {
         sEthernetNetworkCommissioningInstance.Init();
     }
+
+    gChipEventFifoPath = kChipEventFifoPathPrefix + std::to_string(getpid());
+
+    // Creating the named file(FIFO)
+    if ((mkfifo(gChipEventFifoPath.c_str(), 0666) == -1) && (errno != EEXIST))
+    {
+        ChipLogError(Zcl, "Could not create Event FIFO");
+        exit(1);
+    }
```

Comment:
Can we pass the fd return by `mkfifo` info `EventCommandListenerTask`, such that we do not open it again